src/arrange.jl

```diff
@@ -0,0 +1,145 @@
+"""
+    BoundingBox(jpaths::Vector{JPath})
+    
+return BoundingBox enclosing all the jpaths
+"""
+function Luxor.BoundingBox(jpaths::Vector{JPath})
+    allpoints = Point[]
+    for jpath in jpaths
+        for poly in jpath.polys
+            for p in poly
+                push!(allpoints, p)
+            end
+        end
+    end
+    return Luxor.BoundingBox(allpoints)
+end
+
+
+function Luxor.BoundingBox(obj::Object)
+    if isempty(obj.jpaths)
+        @warn "obj.jpaths empty, call getjpaths! on obj first"
+    end
+    return Luxor.BoundingBox(obj.jpaths)
+end
+
+"""
+    center(bbox::Luxor.BoundingBox)
+
+returns the center of the bounding box
+"""
+function center(bbox::Luxor.BoundingBox)
+    bbox.corner1 + (bbox.corner2 - bbox.corner1) / 2.0
+end
+
+"""
+    gtranslate()
+
+global translate independant of any transformations,
+displaces object by a vector specified by `p`
+
+This translation (unlike anim_translate) works by changing the object's start_pos. 
+
+use this if you want to move an object by a certain amount irrespective 
+of any other transforming actions that you apply  to an object.
+
+example
+
+act!(obj1, Action!(1:20,gtranslate( Point(10,20) )))
+"""
+function gtranslate(p::Point)
+    return (v, o, a, f) -> begin
+        if f == first(get_frames(a))
+            a.defs[:now_pos] = o.start_pos
+        end
+        o.start_pos = a.defs[:now_pos] + get_interpolation(a, f) * (p)
+        if f == last(get_frames(a))
+            o.start_pos = a.defs[:now_pos] + p
+        end
+    end
+end
+
+"""
+transform bounding box by m
+"""
+function Base.:*(m::Matrix, b::Luxor.BoundingBox)
+    p1 = Point((m * [b.corner1.x, b.corner1.y, 1])[1:2]...)
+    p2 = Point((m * [b.corner2.x, b.corner2.y, 1])[1:2]...)
+    return Luxor.BoundingBox(p1, p2)
+end
+
+"""
+    transformed_bbox(jpaths, m)
+
+transform the object by matrix m and return its bounding box
+"""
+function transformed_bbox(obj::Object, m::Matrix)
+    ps = Point[]
+    jpaths = obj.jpaths
+    for jpath in jpaths
+        for poly in jpath.polys
+            for p in poly
+                push!(ps, Point((m * [p.x, p.y, 1])[1:2]...))
+            end
+        end
+    end
+    Luxor.BoundingBox(ps)
+end
+
+
+"""
+    arrange()
+
+arranges objects 
+returns a closure to be used with act!(frames,Function)
+
+frames:: the global frames  during which the arrangment should take place
+objects:: array of objects that should be aligned
+p:: Point under/to the side of which arranging should take place
```

Comment:
This can be elaborated further. Maybe take the example of  `0+10`

Reply:
didnt quite catch what you wanted , i forgot maybe i added the example that folows after seeing your comment , is that what you wanted

Reply:
okay it looks like i did add after seeing your comment, couldnt see the entire changes on this page itself . Take a look and unresolv/leave new comment if its not what you were talking about